template.qmd

```diff
@@ -3,6 +3,9 @@ title: Quarto Template for Springer Nature
 format:
   nature-pdf:
     journal: "sn-nature"
```

Comment:
```suggestion
    bib-style: sn-nature
```

It might make more sense to rename this and its sources to `bib-style` or something similar since this is just passed as a class option to the sn-jnl document class to select the bibliography style. I think this would also favor the conversion to natbib as the default cite-method.

Reply:
https://github.com/christopherkenny/nature/pull/2#discussion_r1510564296